docker/registry/python/gradle.properties

```diff
@@ -1,3 +1,3 @@
 io.deephaven.project.ProjectType=DOCKER_REGISTRY
-deephaven.registry.imageName=python:3.7
-deephaven.registry.imageId=python@sha256:6eae21f868957a5d2ae559dc639474482770d37d48d33377de6a69c7e7fb4eab
+deephaven.registry.imageName=python:3.10
+deephaven.registry.imageId=python@sha256:74cdd039dc36f6476dd5dfdbc187830e0c0f760a1bdfc73d186060ef4c4bd78f
```

Comment:
I don't have a problem w/ this; assuming it was necessary for python client testing? I'm assuming this doesn't have anything to do w/ the server itself, b/c we have a min version of 3.8 there.

Reply:
Yes, 3.7 is EOL, minimum requirement is 3.8, so we brought this up to 3.10, same as our server image uses:
https://github.com/deephaven/deephaven-server-docker/blob/1b31ee1ca7686a5e45a448a648714e3cef0fc1cd/server.hcl#L45-L47